apps/src/javalab/JavalabEditor.jsx

```diff
@@ -28,7 +27,9 @@ import {EditorState, Compartment} from '@codemirror/state';
 import {projectChanged} from '@cdo/apps/code-studio/initApp/project';
 import {oneDark} from '@codemirror/theme-one-dark';
 import color from '@cdo/apps/util/color';
+import classNames from 'classnames';
 import {Tab, Nav, NavItem} from 'react-bootstrap';
+import cssModuleStyles from './javalab-editor.module.scss';
```

Comment:
nit: maybe `javalabEditorStyles`? `cssModuleStyles` sounds a bit generic/unrelated to this file?

Reply:
Yeah I didn't know what to call this, I see us using the name `style` for the import from SCSS files in DSCO, which seemed too close to the `styles` object since I didn't fully move to SCSS. The `styles` object seems also like it would represent "javalab editor styles", so I thought I'd distinguish based on the source of the styling. I'll ask in our swarm channel if anyone has thoughts on good names here :).

Reply:
i'm hoping we don't run into this too often -- if we're converting a component to use modules instead of JS styles, we shouldn't have to manage naming collision/confusion. so, my first question is: can we get rid of JS styles in this component (and call the SCSS module import `style`)?

if that's not feasible right now, then i think `moduleStyle` is an appropriate name -- `style` has been our default name, and i removed the "css" prefix so people aren't confused by CSS vs SCSS (🤷🏻‍♀️)

Reply:
> so, my first question is: can we get rid of JS styles in this component (and call the SCSS module import style)?

This was the case I was referring to when I asked about whether we were ok with [only half-converting a component's styles to SCSS](https://codedotorg.slack.com/archives/C03MZ259SUU/p1659570937438829) in order to remove Radium. It's a particularly "tech debt-y" component that does way too much (we're [working on breaking it up!](https://github.com/code-dot-org/code-dot-org/pull/47486)), including providing styling to subcomponents, which makes it time consuming/challenging to figure out what each style is doing in the first place (a lot of looking at git history/devtools/etc).

I guess in general I feel like I can do Radium deprecation work a lot faster if I'm not trying to also figure out why every style in each component we're deprecating exists? Should I be thinking of Radium deprecation and moving to SCSS modules as work that we don't particularly care which gets done first? I had been leaning towards "let's remove Radium from as many components as possible and we'll figure out SCSS later (but try and do SCSS now if its easy)", but not sure if that's right.

Reply:
ah, i didn't realize this was that same case you mentioned before.

> Should I be thinking of Radium deprecation and moving to SCSS modules as work that we don't particularly care which gets done first?

for summer swarms, i think our goal is what you said -- removing radium and working out the rest of our SCSS conversion later. but once swarms end, we need to start moving toward that end goal, so i keep feeling conflicted 🙁 so i've been making case-by-case calls, which in this case i think you're right to wait on the full conversion